src/modules/LanItems.rb

```diff
@@ -2018,21 +2029,24 @@ def Commit
         newdev["INTERFACETYPE"] = @type
       end
 
-      NetworkInterfaces.Name = Ops.get_string(@Items, [@current, "ifcfg"], "")
-      NetworkInterfaces.Current = deep_copy(newdev)
+      if GetCurrentMap() != newdev
```

Comment:
- GetCurrentMap is not documented
- it fetches the ifcfg from `NetworkInterfaces`, not from `Items`. what is the relationship between the two at this point?